runtime/oti/j9nonbuilder.h

```diff
@@ -5346,7 +5346,11 @@ typedef struct J9JavaVM {
 	struct J9HashTable* classLoadingConstraints;
 	UDATA* vTableScratch;
 	UDATA vTableScratchSize;
+#if defined(J9VM_JIT_CLASS_UNLOAD_RWMONITOR)
 	omrthread_rwmutex_t classUnloadMutex;
+#else
+	omrthread_monitor_t classUnloadMutex;
+#endif
```

Comment:
What does this have to do with OSX?

Reply:
I was building without the JIT and the compiler complained about the incorrect type in some of the calls.  So not strictly OSX related, but found as part of this work.

Reply:
ok